editor/src/components/editor/editor-component.tsx

```diff
@@ -555,14 +564,32 @@ const LockedOverlay = React.memo(() => {
         left: leftMenuExpanded ? LeftPaneDefaultWidth : 0,
         width: '100vw',
         height: '100vh',
-        backgroundColor: '#000',
+        backgroundColor: '#00000044',
         zIndex: 30,
-        opacity: 0.2,
         transition: 'all .1s ease-in-out',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
       }}
       css={css`
         animation: ${anim} 0.3s ease-in-out;
       `}
-    />
+    >
+      {refreshingDependencies && (
```

Comment:
Please use the `when` utility function instead.

Reply:
you're right, sorry, damn muscular memory